tests/FSharp.AWS.DynamoDB.Tests/SimpleTableOperationTests.fs

```diff
@@ -272,10 +309,17 @@ type ``TransactWriteItems tests``(fixture: TableFixture) =
     }
 
     [<Fact>]
-    let ``Empty request list is rejected with AORE`` () = shouldBeRejectedWithArgumentOutOfRangeException []
+    let ``Empty request list is rejected with AORE`` () =
+        shouldBeRejectedWithArgumentOutOfRangeException (Transaction())
+        |> Async.RunSynchronously
+        |> ignore
```

Comment:
is this `ignore` needed? you can make the signature on L312 `: unit` as a safeguard